noxfile.py

```diff
@@ -96,7 +99,7 @@ def pytest_args(*args):
 
 def run_pytest(session, directory=".", *args):
     pytest_cmd = pytest_args(directory, *args)
-    session.run(*pytest_cmd, silent=SILENT)
+    session.run(*pytest_cmd, silent=PYTEST_SILENT)
```

Comment:
I think this is enough for now, no need for the rest.
put a comment next to it explaining why.

```suggestion
    session.run(*pytest_cmd, silent=False)
```

Reply:
Sure. I thought in your earlier comment you meant only changing it for circleCI tests, guess i misunderstood that :)